manifests/n/Nicotine+/Nicotine+/3.3.4/Nicotine+.Nicotine+.locale.en-US.yaml

```diff
@@ -0,0 +1,32 @@
+# Created with komac v2.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Nicotine+.Nicotine+
+PackageVersion: 3.3.4
+PackageLocale: en-US
+Publisher: Nicotine+
+PublisherUrl: https://nicotine-plus.org/
+Author: Avery Terrel
+PackageName: Nicotine+
+PackageUrl: https://nicotine-plus.org/
+License: GPL-3.0
+LicenseUrl: https://github.com/nicotine-plus/nicotine-plus/blob/master/COPYING
+Copyright: Copyright (C) 2007 Free Software Foundation, Inc. <https://fsf.org/>
+CopyrightUrl: https://github.com/nicotine-plus/nicotine-plus/blob/master/COPYING
+ShortDescription: Graphical client for the Soulseek peer-to-peer network
+Description: Nicotine+ aims to be a lightweight, pleasant, free and open source (FOSS) alternative to the official Soulseek client, while also providing a comprehensive set of features.
+Moniker: Nicotine+
+Tags:
+- chat
+- files
+- gtk
+- gtk3
+- gtk4
+- messaging
+- p2p
+- peer-to-peer
+- python
+- soulseek
+ReleaseNotesUrl: https://github.com/nicotine-plus/nicotine-plus/releases/tag/3.3.4

```

Comment:
@averyterrel For future reference the actual Release Notes file for end users is at link:

https://nicotine-plus.org/NEWS.html